src/Pos/Update/Poll/RollTrans.hs

```diff
@@ -72,13 +73,22 @@ instance MonadPoll m => MonadPoll (RollT m) where
     delConfirmedProposal = lift . delConfirmedProposal
 
     addActiveProposal ps = RollT $ do
+        whenNothingM_ (use unPrevProposersL) $ do
+            prev <- getEpochProposers
+            unPrevProposersL .= Just prev
         insertIfNotExist (hash $ psProposal $ ps) unChangedPropsL getProposal
         addActiveProposal ps
 
     deactivateProposal id = RollT $ do
         insertIfNotExist id unChangedPropsL getProposal
```

Comment:
no, actually we can't determine that we actually deactivate proposal which was proposed in the current epoch or before (without some addition information).

so I decided to use next contract: if proposer proposed new proposal, he can't propose new proposals in current epoch in any case (even if we deactivate this proposal in current epoch)